web_src/js/features/imagediff.js

```diff
@@ -88,29 +91,27 @@ export function initImageDiff() {
 
     for (const info of imageInfos) {
       if (info.$image.length > 0) {
-        $.ajax({
-          url: info.path,
-          success: (data, _, jqXHR) => {
-            info.$image.on('load', () => {
-              info.loaded = true;
-              setReadyIfLoaded();
-            }).on('error', () => {
-              info.loaded = true;
-              setReadyIfLoaded();
-              info.$boundsInfo.text('(image error)');
-            });
-            info.$image.attr('src', info.path);
+        info.$image.on('load', () => {
+          info.loaded = true;
+          setReadyIfLoaded();
+        }).on('error', () => {
+          info.loaded = true;
+          setReadyIfLoaded();
+          info.$boundsInfo.text('(image error)');
+        });
+        info.$image.attr('src', info.path);
 
-            if (jqXHR.getResponseHeader('Content-Type') === 'image/svg+xml') {
-              const bounds = getDefaultSvgBoundsIfUndefined(data, info.path);
-              if (bounds) {
-                info.$image.attr('width', bounds.width);
-                info.$image.attr('height', bounds.height);
-                hideElem(info.$boundsInfo);
-              }
-            }
+        // this may be dead code as we currently do not render SVGs images in image diffs
```

Comment:
No, we do render SVG. I always use SVG to test, like #26696

I guess this PR stops the SVG from working.

Reply:
It does show for me, at least like #26696

Reply:
Can you link to a try example where it shows? Maybe it only shows in pull request diffs?

Reply:
Ah I see the problem, I had svg renderer disabled: https://github.com/go-gitea/gitea/blob/393193a3e075b2dc51bbd43a1cf440450f97907d/custom/conf/app.example.ini#L1296

Reply:
Removed the comment and did a few checks that `getDefaultSvgBoundsIfUndefined` works as before and it seems good, it correctly scaled a SVG with percentage width. Overall I think this JS can probably be replaced with some CSS but I'll not dig too deeply now into this.